src/schemas.py

```diff
@@ -1,10 +1,33 @@
 import json
 from datetime import datetime
+from enum import Enum
 from typing import Optional
 
 from src.models import CustomModel, Product, RedisEvent
 
 
+class Product(Enum):
```

Comment:
to my knowledge this should already be in `models.py`? Any reason we are adding it here?

Reply:
tbh not sure, @JeffersonDing ^?